### ISSUE TITLE
Switch from Column Level mode to Table Level Mode in UI

### ISSUE THREAD

**Issue by YLibert** [MarquezProject/marquez#2792]
It's not currently possible to switch from a Column Level Mode 

![column_level_lineage](https://github.com/MarquezProject/marquez/assets/60961333/4812589c-e065-4840-a1f4-98ebc1ce1d60)

To a Table Level Mode 

![table_level_lineage](https://github.com/MarquezProject/marquez/assets/60961333/318afefc-8f54-49a3-b585-24c40c6b172b)

In the Lineage View. 

It would be nice to be able to choose between the two modes

**Comment by wslulciuc** [comment]
Agreed! As we revisit our lineage model (and queries), we’ll make sure `in`/`out` edges of the graph can be either: `datasets`, `jobs`, and `columns`; this will unify the lineage graph data making what you're suggesting possible.
